nilearn/image/tests/test_image.py

```diff
@@ -582,6 +582,10 @@ def test_clean_img():
     data_img_nifti2_ = image.clean_img(
         data_img_nifti2, detrend=True, standardize=False, low_pass=0.1, t_r=1.0)
 
+    # if mask_img
+    img, mask_img = data_gen.generate_fake_fmri(shape=(10, 10, 10), length=10)
+    data_img_mask_ = image.clean_img(img, mask_img=mask_img)
```

Comment:
You may add some checks on the result ?

Reply:
I didn't add any checks on the result, as the other tests in this section also didn't check for results. But I could verify that a `np.ones(...)` mask and no mask would get the same result?

Reply:
Indeed, it would amount to testing that apply_mask is doing its job, which is not what we need here.